apps/docs/src/guide/encoding/encode-and-decode.md

```diff
@@ -0,0 +1,21 @@
+# Encode and Decode
+
+Encoding is the process of transforming raw data into bytecode as per a set of rules. Decoding is the reverse process, transforming bytecode back to raw data. The SDK provides the `AbiCoder` class to encode and decode data.
```

Comment:
I think it's more appropriate to outline how encoding and decoding is specific to our SDK as opposed to discussing what that is in a broad sense, given encoding/decoding isn't unique to our ecosystem. 

```suggestion
In order to interact with the FuelVM,  our types must be encoded via according to the [ABI spec](https://docs.fuel.network/docs/specs/abi/). Subsequently, we must transform this bytecode in order to use it in our SDK. To encode a type, it is first converted to an [`JsonAbiType`](https://docs.fuel.network/docs/specs/abi/argument-encoding/#types). To decode a type, you also need to provide a describing the schema of the type in question. All types generated via our 
[ABI Typegen package](https://docs.fuel.network/docs/fuels-ts/fuels-cli/abi-typegen/#abi-typegen)
 ```

Reply:
Sounds good, I've made a few [changes](https://github.com/FuelLabs/fuels-ts/pull/2405/commits/afc7691126b4867187408fa2616a6f2bc56b1fb1).